sdk/digitaltwins/digitaltwins/src/digitalTwinsClient.ts

```diff
@@ -0,0 +1,851 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  TokenCredential,
+  ServiceClientCredentials,
+  RestResponse,
+  RequestOptionsBase
+} from "@azure/core-http";
+import { PageSettings, PagedAsyncIterableIterator } from "@azure/core-paging";
+import { v4 } from "uuid";
+import { AzureDigitalTwinsAPI as GeneratedClient } from "./generated/src/azureDigitalTwinsAPI";
+import {
+  AzureDigitalTwinsAPIOptions,
+  DigitalTwinsGetByIdResponse,
+  DigitalTwinsAddOptionalParams,
+  DigitalTwinsAddResponse,
+  DigitalTwinsUpdateOptionalParams,
+  DigitalTwinsUpdateResponse,
+  DigitalTwinsDeleteMethodOptionalParams,
+  DigitalTwinsGetComponentResponse,
+  DigitalTwinsUpdateComponentResponse,
+  DigitalTwinsUpdateComponentOptionalParams,
+  DigitalTwinsAddRelationshipResponse,
+  DigitalTwinsAddRelationshipOptionalParams,
+  DigitalTwinsUpdateRelationshipOptionalParams,
+  DigitalTwinsUpdateRelationshipResponse,
+  DigitalTwinsDeleteRelationshipOptionalParams,
+  DigitalTwinsListIncomingRelationshipsResponse,
+  DigitalTwinsListRelationshipsOptionalParams,
+  DigitalTwinsListRelationshipsResponse,
+  DigitalTwinsSendTelemetryOptionalParams,
+  DigitalTwinsSendComponentTelemetryOptionalParams,
+  DigitalTwinModelsGetByIdResponse,
+  DigitalTwinModelsListOptionalParams,
+  DigitalTwinModelsListResponse,
+  DigitalTwinModelsListOptions,
+  DigitalTwinModelsAddResponse,
+  DigitalTwinModelsAddOptionalParams,
+  ModelData,
+  IncomingRelationship,
+  EventRoutesGetByIdResponse,
+  EventRoutesListOptionalParams,
+  EventRoutesListNextResponse,
+  EventRoute,
+  EventRoutesAddOptionalParams,
+  QueryQueryTwinsResponse,
+  QuerySpecification
+} from "./generated/src/models";
+
+/**
+ * Client for Azure IoT DigitalTwins API.
+ */
+export class DigitalTwinsClient {
+  /**
+   * @internal
+   * @ignore
+   * A reference to the auto-generated AzureDigitalTwinsAPI
+   */
+  private readonly client: GeneratedClient;
+
+  // private readonly client!: AzureDigitalTwinsAPI;
+
+  /**
+   * Creates an instance of AzureDigitalTwinsAPI.
+   *
+   * Example usage:
+   * ```ts
+   * const { DigitalTwinsClient, ServiceClientCredentials } = require("@azure/digitaltwins");
+   *
+   * const client = new DigitalTwinsClient(
+   *   "<endpoint>",
+   *   new DefaultAzureCredential();
+   * );
+   * ```
+   * @param {string} endpoint The endpoint of the service. If options parameter is given than this value will be ignored.
+   * @param {TokenCredential | ServiceClientCredentials} credential Used to authenticate requests to the service.
+   * @param {AzureDigitalTwinsAPIOptions} [options] Used to configure the service client.
+   */
+  constructor(
+    enpoint: string,
+    credential: TokenCredential | ServiceClientCredentials,
+    options?: AzureDigitalTwinsAPIOptions
```

Comment:
put the default in the signature

Reply:
Fixed

---
In reply to: [447259021](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447259021) [](ancestors = 447259021)